packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -314,9 +327,30 @@ public function import_term( $data ) {
 
 				// Wipe the parent for now
 				$data['parent'] = 0;
+			}*/
+			$parent_term = term_exists( $parent, $data['taxonomy'] );
```

Comment:
What's the performance impact of running a SELECT before every insert on large databases?